week-7/Homework/mandatory/0-thinking-like-a-programmer/task.md

```diff
@@ -27,11 +27,12 @@ After you've watched these videos I'd like you to answer these questions
 ## 1. What do you think the most important quality for a programmer is?
 
 <!-- Write your answer here -->
+Problem Solving
 
 ## 2. When trying to solve a challenge, what should you do first?
-
+try to process the problem bit by bit, take small part and solve it then take another part and so on untill you finish
 <!-- Write your answer here -->
 
 ## 3. What should you do if you get stuck?
-
+seek help, look online, ask a peer or a mentor. If non could help refer to documentation.
```

Comment:
Yep, documentation is a good point especially when we need specific details. Although it is tedious to read sometimes, it should be the source of truth.

Reply:
Seek for help in the right time and not getting stuck too long in the rabbit hole.